packages/mobile/src/sentry/Sentry.ts

```diff
@@ -11,7 +11,8 @@ const TAG = 'sentry/Sentry'
 
 // Set this to true, if you want to test Sentry on dev builds
 // Set tracesSampleRate: 1 to capture all events for testing performance metrics in Sentry
-export const SENTRY_ENABLED = !__DEV__ || false
+// Set SENTRY_ENABLED to false during e2e tests
+export const SENTRY_ENABLED = (!isE2EEnv && !__DEV__) || false
```

Comment:
sorry i know we talked about this but now that i stare at it....not sure why we needed the `|| false` before (`true || false`/`false || false` seems redundant 😕  )

Reply:
My reasoning here is that e2e tests run in release mode and we want to enable sentry on release modes that are not e2e runs.

Reply:
I'm :-1: on implicit behavior like this.  I.e., isE2EEnv implies that sentry is disabled.  It's not obvious to others what the entire implied behavior of isE2EEnv is.  I'd rather have a config value of SENTRY_ENABLED (i.e., in src/config.ts or whatever) that we explicitly set.

Reply:
Currently e2e tests run in release mode on Alfajores, but we could in the future run e2e tests on mainnet. Since we cannot base SENTRY_ENABLED solely on the network or release v.s. dev mode. It might be best to explicitly set it in the run_e2e.sh

Reply:
+1 to explicitly setting it and avoiding intermixing configuration with code via magic config variables like `isE2EEnv`.